NETDefault/Gateway.Buckaroo/ReadMe.md

```diff
@@ -0,0 +1,15 @@
+﻿# Buckaroo kickstart
+
+[Buckaroo](https://www.buckaroo.nl/) is a Payment Service Provider (PSP) from Dutch origin which is on the market for quite some time already. In the &C project (june 2022), we've used this provider for the registration of customers and their subscriptions, and the automatic billing - using the '[Subscription](https://support.buckaroo.nl/categorieen/subscriptions?mark=Subscriptions)' and '[Credit Management](https://support.buckaroo.nl/categorieen/credit-management?mark=credit%20management)' parts of Buckaroo respectively. The .NET SDK has not been used.
+
+The code in this project functions as a kickstart to get you up and running faster. It contains the HTTP Client to be injected which will be sending requests using [HMAC](https://wiki.wearetriple.com/pages/viewpage.action?spaceKey=GEN&title=Office+VPN). Also included is an example request to [GET a debtor](https://dev.buckaroo.nl/AdditionalServices/Description/creditmanagement#debtorinfo) (a.k.a. customer).

```

Comment:
I don't want internal urls in public documentation, so remove wiki.wearetriple.com.

Reply:
My bad, wrong link.